.pylintrc

```diff
@@ -80,7 +80,7 @@ persistent=yes
 
 # Minimum Python version to use for version dependent checks. Will default to
 # the version used to run pylint.
-py-version=3.8
+py-version=3.10
```

Comment:
I don't think we should change this to 3.10. The minimum version should still be 3.8. All this PR should be doing is making sure `dynamicio` is compatible with `3.9` and `3.10`. We can set this up see here: https://yellowdesert.consulting/2020/10/24/tox-testing-multiple-python-versions-with-pyenv/ 

It will impact the ci duration and we don't have any ci setup at the moment which is a good idea to address in this PR. I can support this task.

Reply:
Let's do it! 👍